### ISSUE TITLE
Analysis Exception introduced with Python3 (maximum recursion depth exceeded in comparison)

### ISSUE THREAD

**Issue by Ana06** [vivisect/vivisect#358]
Vivisect doesn't extract the same features for this file with the current master (Python 3) and with vivisect 0.2.0 (Python 2):
https://github.com/fireeye/capa-testfiles/blob/master/kernel32.dll_

I think it is caused by the following exception I see in the logs:
```
INFO:vivisect:Function Analysis Exception for function 0x7de00e11, module: vivisect.analysis.i386.calling
INFO:vivisect:Exception Traceback: Traceback (most recent call last):
  File "/path/vivisect/__init__.py", line 781, in analyzeFunction
    fmod.analyzeFunction(self, fva)
  File "/path/vivisect/analysis/i386/calling.py", line 142, in analyzeFunction
    emumon.addAnalysisResults(vw, emu)
  File "/path/vivisect/impemu/monitor.py", line 112, in addAnalysisResults
    reprargs = [emu.reprVivValue(val) for val in argv]
  File "/path/vivisect/impemu/monitor.py", line 112, in <listcomp>
    reprargs = [emu.reprVivValue(val) for val in argv]
  File "/path/vivisect/impemu/emulator.py", line 578, in reprVivValue
    return self.reprVivTaint(taint)
  File "/path/vivisect/impemu/emulator.py", line 548, in reprVivTaint
    argsstr = ','.join([self.reprVivValue(x) for x in argv])
  File "/path/vivisect/impemu/emulator.py", line 548, in <listcomp>
    argsstr = ','.join([self.reprVivValue(x) for x in argv])
  File "/path/vivisect/impemu/emulator.py", line 578, in reprVivValue

[...]

  File "/path/vivisect/impemu/emulator.py", line 578, in reprVivValue
    return self.reprVivTaint(taint)
  File "/path/vivisect/impemu/emulator.py", line 547, in reprVivTaint
    callstr = self.reprVivValue(pc)
  File "/path/vivisect/impemu/emulator.py", line 571, in reprVivValue
    taint = self.getVivTaint(val)
  File "/path/vivisect/impemu/emulator.py", line 510, in getVivTaint
    return self.taints.get( va & self.taintmask )
RecursionError: maximum recursion depth exceeded in comparison
```

### Steps to reproduce

```
> python 3
>>> # Enable logging
>>> import logging
>>> logging.basicConfig(level=logging.INFO)
>>>
>>> import vivisect
>>> file = 'kernel32.dll_' # https://github.com/fireeye/capa-testfiles/blob/master/kernel32.dll_
>>> vw = vivisect.VivWorkspace()
>>> vw.loadFromFile(file)
>>> vw.analyze() # Exception! 
```

**Comment by rakuy0** [comment]
Cool. I'm triaging this today. I'm naively hopeful my cleanup branch will address this, but I have nothing to base that on, but I will update this once I get a chance to run it.

**Comment by Ana06** [comment]
I have just tried master + Py3 Cleanup (#353) + Python3 fixes (#357) and this still happens.

**Comment by rakuy0** [comment]
Yea, I saw the same this morning when I tried it. Unfortunate, but I wasn't confident it would fix it. I'll dig deeper into this once I get some of the other bugfix branches merged.

**Comment by rakuy0** [comment]
Oh, as with the other ticket, can you be more specific about what features are missing? That way I know what to look for once I squash this bug and can confirm things are working.

**Comment by Ana06** [comment]
I have just checked again, and it seems that the feature extraction problem is solved in master + Py3 Cleanup (#353) + Python3 fixes (#357). This also means that this was not related to the exception above, because it still happens. Sorry, I should have checked both things.
